src/jit/optcse.cpp

```diff
@@ -748,6 +765,102 @@ unsigned Compiler::optValnumCSE_Locate()
     return 1;
 }
 
+//------------------------------------------------------------------------
+// updateCseArrLenMap: Check if this compare is a tractable function of
+//                     an array length that is a CSE candidate, and insert
+//                     an entry in the cseArrLenMap if so.  This facilitates
+//                     subsequently updating the compare's value number if
+//                     the array length gets CSEd.
+//
+// Arguments:
+//    compare - The compare node to check
+
+void Compiler::updateCseArrLenMap(GenTreePtr compare)
```

Comment:
I'm not a fan of such prefixes but considering that `Compiler` has a zillion members maybe it's better to prefix stuff. The whole thing looks like a ton of C code dumped into a C++ class...